backend/routes/adminRoutes.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const router = express.Router();
+
```

Comment:
You can add here a middleware for admin authentication to ensure only admins access this route.

```javascript
// Middleware for admin authentication
const adminAuthMiddleware = (req, res, next) => {
  const isAdmin = true; // Replace with actual admin authentication logic
  if (!isAdmin) {
    return res.status(403).send({ error: 'Forbidden: Admin access only' });
  }
  next();
};

router.use(adminAuthMiddleware); // Protect all admin routes
```